src/utils/triangle.ts

```diff
@@ -0,0 +1,15 @@
+const getTrianglePerimetr = (a: number, b: number, c: number): number =>
+  a + b + c;
```

Comment:
[CODE STYLE] - The function name `getTrianglePerimetr` contains a typo. The correct spelling should be `getTrianglePerimeter`. Always ensure that function names are spelled correctly to avoid confusion and maintain code readability.